packages/components/src/FormatFile/FormatFile.css

```diff
@@ -69,10 +89,49 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
 .contentBlock p {
   margin: 0;
 }
 
 .actionBlock {
   padding: 0 var(--file-card--base-padding) 0 0;
 }
+
+.thumbnail {
+  --message-weight: 400;
+  display: flex;
+  display: inline-block;
+  position: relative;
+  height: 50px;
+  font-weight: var(--message-weight);
+  align-items: center;
+}
+
+.deleteButton {
+  position: absolute;
+  top: var(--space-smaller);
+  right: var(--space-smaller);
+  z-index: 1;
+}
+
+.bold {
+  --message-weight: 800;
+  transform: uppercase;
+}
+
+.centered {
+  display: flex;
+  position: relative;
+  height: 100%;
+  margin: 0 var(--space-small);
+  align-items: center;
+}
+
+.overlay {
```

Comment:
Reference comment above about this overlay conflicting with the original technique. Same overlay can be used for the progress bar and the delete action.